docs/build/build-open-source.md

```diff
@@ -80,7 +67,7 @@ In the below sections you can find a list of different layers of the Polkadot St
 | Components | Existing projects | Potentially interesting projects
 |-|-|-
 | Desktop/Web Wallets | [Talisman Web Application](https://github.com/TalismanSociety/talisman-web) :green_heart:, [AirGap](https://github.com/airgap-it/airgap-wallet) :green_heart:, [Sakura](https://github.com/w3finance/sakura) :broken_heart:| User-friendly Wallet based on the [Recovery Pallet](https://github.com/paritytech/substrate/tree/master/frame/recovery)<!-- NO_STATUS_BADGE -->, Web wallets focused on user-onboarding (e.g. using [localStorage](https://github.com/near/near-wallet)<!-- NO_STATUS_BADGE --> )
-| Browser Extensions | [Polkadot{.js}](https://github.com/polkadot-js/extension) :green_heart:, [Polkadot-Js-Plus-Extension](https://github.com/Nick-1979/polkadot-Js-Plus-extension) :green_heart:, [SubWallet-Extension](https://github.com/Koniverse/SubWallet-Extension) :green_heart:, [Doter](https://github.com/ChainBridgeNetworkTeam/Doter) :broken_heart:, [Enzyme](https://github.com/blockxlabs/enzyme/) :broken_heart:, [Speckle OS](https://github.com/GetSpeckle/speckle-browser-extension) :broken_heart:| Sign-in with your polkadot, kusama, etc. account.  
+| Browser Extensions | [Enkrypt](https://github.com/enkryptcom/enKrypt), [Polkadot{.js}](https://github.com/polkadot-js/extension) :green_heart:, [Polkadot-Js-Plus-Extension](https://github.com/Nick-1979/polkadot-Js-Plus-extension) :green_heart:, [SubWallet-Extension](https://github.com/Koniverse/SubWallet-Extension) :green_heart:, [Doter](https://github.com/ChainBridgeNetworkTeam/Doter) :broken_heart:, [Enzyme](https://github.com/blockxlabs/enzyme/) :broken_heart:, [Speckle OS](https://github.com/GetSpeckle/speckle-browser-extension) :broken_heart:| Sign-in with your polkadot, kusama, etc. account.  
```

Comment:
@lucasvanmol Can you confirm this is true?

Reply:
For us it worked without :grey_question: or anything similar.

Reply:
As @Noc2 said, you don't need to append anything. Since the option `find-all-links` is true, the only thing that matters is if you include `<-- NO_STATUS_BADGE -->` after a link, it won't place any badge there.